src/components/Login/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  ModalFooter,
+} from "@chakra-ui/react";
+import { Flex, Text, Icon, Link, useDisclosure } from "@chakra-ui/react";
+
+import GoogleSvg from "@/assets/svg/LoginSvg/gmail.svg";
+import EmailSvg from "@/assets/svg/LoginSvg/email.svg";
+
+function LoginModalContent(props: any) {
+  return (
+    <Modal isOpen={props.isOpen} onClose={props.onClose} isCentered size={"sm"}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Flex flexDir="column" mt="40px" mb="20px">
+            <Text fontSize="xl" fontWeight="bold">
+              Log in or sign up
+            </Text>
+            <Text fontSize="sm">Use your email or google account to continue with CourtCanva</Text>
+          </Flex>
+        </ModalHeader>
+        <ModalCloseButton />
+        <ModalBody mb="20px">
+          <Flex flexDir="column" justifyContent="space-around" gap="25px">
+            <Button variant="loginBtn2" position="relative">
+              <Icon w="24px" h="24px" position="absolute" top="11px" left="20px">
+                <GoogleSvg />
+              </Icon>
+              <Text>Continue with Google </Text>
+            </Button>
+
+            <Button variant="loginBtn2" position="relative">
+              <Icon w="24px" h="24px" position="absolute" top="12px" left="20px">
+                <EmailSvg />
```

Comment:
<img width="364" alt="image" src="https://user-images.githubusercontent.com/61776449/171647770-289d8cfd-d89c-4900-a15e-4df0968497b6.png">
White icon with gray background, this contrast is strange.
Do we have any design for this login page?

Reply:
Thanks Robert! I have fixed all of it.